processors/src/claim-processor/handler.js

```diff
@@ -81,6 +81,11 @@ class ClaimHandler extends TransactionHandler {
    */
   async deleteClaim({ data: claim }, state) {
     const { vin } = claim.vehicle;
+    const existingClaim = await state.getClaim(vin);
```

Comment:
Why is this necessary?

This exact same thing happens the way that it is currently written.

Reply:
Without this, trying to delete a nonexistent claim doesn't do anything (which is fine) but just returns "undefined", the same message that's returned when a claim is deleted.  This way we at least get an error message returned.

Reply:
Why should it even be an error. Who cares if it doesn't exist. The same result occurs either way -> claim doesn't exist.

Reply:
Also, most `DELETE` requests don't return a body. I'm not aware of a single api that does actually.